src/helpers.ts

```diff
@@ -0,0 +1,23 @@
+export class Helpers {
+    private token: string
+
+    constructor (token: string) {
+        this.token = token
+    }
+
+    async downloadArtifactFromURL (url: string): Promise<ArrayBuffer> {
+        if (!url.startsWith('https://')) {
```

Comment:
I suggest we assert the domain name as well. AFAIK it's always circleci-artifacts.com or something like that.